pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1470,5 +1471,10 @@ def _check_plot_works(f, freq=None, series=None, *args, **kwargs):
 
         with ensure_clean(return_filelike=True) as path:
             plt.savefig(path)
+
+        # GH18439
```

Comment:
why is this only PY3?

add a more informative comment, not just the issue, e.g. ensure round-trip pickle compat

Reply:
@Licht-T saw your comment, ok then, just make a comment that we are pickling instancemethods which is supported on PY3 only (hmm thought this did work on PY2)